docs/Drawing.md

```diff
@@ -19,4 +19,37 @@ using (var image = new MagickImage(new MagickColor("#ff00ff"), 512, 128))
       .Ellipse(256, 96, 192, 8, 0, 360)
       .Draw(image);
 }
-```
\ No newline at end of file
+```
+
+## Adding Text To Existing Image
+
+![Example of adding text to existing image](./img/example_addTextToExistingImage.jpg)
```

Comment:
This could be the image in the sample directory.

Reply:
👍